.gitignore

```diff
@@ -59,3 +59,6 @@ dist/*
 repos.txt
 oldRelease.json
 newRelease.json
+
+# Ignore golang's vendored files
+vendor/
```

Comment:
Let's make this an absolute path, 
```
/vendor/
```

we have a folder with some testdata named "vendor"
```
$ find -name vendor
./checks/raw/testdata/vendor
```

Reply:
And probably our other go.mod folder too then.
```
/tools/vendor/
```